client/landing/stepper/declarative-flow/internals/steps-repository/unified-plans/index.tsx

```diff
@@ -59,6 +71,40 @@ export default function PlansStepAdaptor( props: StepProps ) {
 		setPlanInterval( intervalType );
 	};
 
+	/**
+	 *  Plan step switches the selected theme to default twentytwentyfour when the plan is free
+	 *  but the selected design requires a paid plan.
+	 */
+	const switchPaidDesignToDefault = ( stepInfo: ProvidedDependencies ) => {
+		const hasPaidDesign =
+			( selectedDesign?.design_tier && selectedDesign?.design_tier !== 'free' ) || false;
```

Comment:
```suggestion
			Boolean( selectedDesign?.design_tier && selectedDesign?.design_tier !== 'free' )
```

nit: might be a bit more clearer than `|| false`